flashcards-ui/src/components/FlashcardComponents/StudyPageContent/StudyPageContent.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Heading, Button, Flex, Center } from "@chakra-ui/react";
+import Index from "./Carousel/Index";
+
+import { useFlashcardContext } from "../../../../contexts/flashcard";
+
+export default function StudyPageContent({ chosenSet }) {
+    let filteredFlashcard = chosenSet.flashcards.filter((e) => {
+        if (e.visibility == true) {
+            return e;
+        }
+    });
+
+    if (filteredFlashcard.length < 2) {
+        console.error(
+            `I don't know how but less than two flashcards are inside of filteredFlashcard, all flashcards will be used`
+        );
+        // console.log(filteredFlashcard)
```

Comment:
RECOMMENDED: Use Javascript Console Utils (VSCode Extension) to add and remove console logs easily.

Reply:
That's just a placeholder for an alert system, should I still remove it?